src/anomalib/models/uflow/anomaly_map.py

```diff
@@ -0,0 +1,160 @@
+"""UFlow Anomaly Map Generator Implementation."""
+
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+from typing import List
+
+import numpy as np
+import scipy.integrate as integrate
```

Comment:
as per the codacy comment,
```suggestion
from scipy import integrate
```